packages/action/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: 'Post TAP Files'
+description: 'Greet someone and record the time'
+inputs:
+  github:
+    description: 'The Github Context JSON object'
```

Comment:
I think most of this information we need is populated in environment variables, see:

https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

So, the user shouldn't need to provide this information.

Reply:
The environment variables are used, but I think they have to be used in the .github/workflows file, and this action.yaml metadata file just passes these environment variables into the javascript